src/api/services/slack_notify/notification.py

```diff
@@ -0,0 +1,44 @@
+import os
+import logging
+
+from slackclient import SlackClient
+
+
+class SlackNotification(object):
+
+    def __init__(self):
+        slack_token = os.getenv('SLACK_API_TOKEN')
+        if slack_token:
+            self.sc = SlackClient(slack_token)
+
+
+
+    def get_slack_id(self, user_email):
+        user_email = user_email
+        results = self.sc.api_call("users.list")
```

Comment:
Is it possible to initialize this as a class variable? It's quite expensive having to get all users with every get_slack_id() call.

Reply:
`get_slack_id()` is called once, returning all users in slack. Single users are returned in the loop below. I will instead call `get_slack_id()` outside the `get_users` loop. Thanks!